pkg/cosign/cosign.go

```diff
@@ -437,8 +438,11 @@ func LoadVerifiers(ctx context.Context, keyRef string, algorithm string) (verifi
 		}
 
 		verifiers = append(verifiers, ghVerifiers...)
+	} else if regexp.MustCompile(`^[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*$`).MatchString(keyRef) {
+		// weak (not length-limited) regexp for github/acorn-manager usernames -> default to acorn manager
```

Comment:
Not sure if there's more to this line that just verifying github usernames exactly, there is a `go-github` repository where they might have a nice function that's maintained with their naming logic you could use. I didn't find one in a cursory look over it but if you want to take a look https://pkg.go.dev/github.com/google/go-github/v54/github#section-documentation

Reply:
Yeah, I didn't bother searching for a 100% matching solution here, just wanted a weak check here to not include cases which definitely not fall into this category.